packages/mobile-top-nav/src/assets/img/icon_audio.js

```diff
@@ -0,0 +1,29 @@
+import { html } from 'lit-element';
+import Icon from './icon';
```

Comment:
The naming scheme for these files should match the component name (using dashes instead of underscore)